app/products/calls/utils.ts

```diff
@@ -150,3 +152,7 @@ export function makeCallsTheme(theme: Theme): CallsTheme {
 
     return newTheme;
 }
+
+export function isCallsStartedMessage(message = '') {
+    return (message === 'You\'ve been invited to a call' || callsMessageRegex.test(message));
+}
```

Comment:
Can we leave a comment to deprecate this at some point?

Reply:
Done in https://github.com/mattermost/mattermost-mobile/pull/7623/commits/13d1946c4cfb00dace6a74da474f9bf068d35b7e